pkg/cli/kubernetes/kubernetes.go

```diff
@@ -20,6 +20,10 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/radius-project/radius/pkg/cli/helm"
+	errors2 "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Can you call this alias `apierrors` I think that's what we call it in a few other places. 

Other ideas are fine too, but `errors2` isn't very readable 👍